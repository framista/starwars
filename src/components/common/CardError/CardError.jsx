import React from 'react';

export const CardError = () => {
  return (
    <div className="w-[500px] max-w-[90%] bg-white rounded shadow-2xl">
      <div className="h-[200px] bg-gray-200 rounded-tr rounded-tl flex justify-center items-center">
        <p className="text-red-500 font-bold">Unexpected problem</p>
      </div>
    </div>
  );
};

export default React.memo(CardError);
