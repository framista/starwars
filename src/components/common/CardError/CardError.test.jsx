import { render } from '@testing-library/react';

import { CardError } from './CardError';

describe('CardError', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<CardError />);
    expect(asFragment()).toMatchSnapshot();
  });
});
