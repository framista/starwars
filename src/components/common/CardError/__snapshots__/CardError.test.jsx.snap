// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardError should render correctly 1`] = `
<DocumentFragment>
  <div
    class="w-[500px] max-w-[90%] bg-white rounded shadow-2xl"
  >
    <div
      class="h-[200px] bg-gray-200 rounded-tr rounded-tl flex justify-center items-center"
    >
      <p
        class="text-red-500 font-bold"
      >
        Unexpected problem
      </p>
    </div>
  </div>
</DocumentFragment>
`;
